.github/workflows/test_install.yml

```diff
@@ -22,17 +22,14 @@ jobs:
 
     - run: echo "AMICI_DIR=$(pwd)" >> $GITHUB_ENV
 
-    # install amici dependencies
+    - name: Install apt dependencies
+      uses: ./.github/actions/install-apt-dependencies
+
     - name: apt
       run: |
         sudo apt-get update \
```

Comment:
Skip the update here?

Reply:
Takes only about 1s to run it a second time, I'd leave it here to keep it's safer to shuffle those steps around.